rs/index/src/ivf/files/invalidated_ids.rs

```diff
@@ -12,13 +12,23 @@ pub struct InvalidatedIdsStorage {
     files: Vec<File>,
     current_backing_id: i32,
     current_offset: usize,
-    backing_id_offset: usize,
+}
+
+pub struct InvalidatedId {
+    pub user_id: u128,
+    pub doc_id: u128,
+}
+
+pub struct InvalidatedIdsIterator {
+    files: Vec<File>,
+    current_file_idx: usize,
+    current_offset: usize,
 }
 
 impl InvalidatedIdsStorage {
     const DEFAULT_BACKING_FILE_SIZE: usize = 8192;
     pub fn new(base_directory: &str, backing_file_size: usize) -> Self {
-        let bytes_per_invalidation = size_of::<u128>() + size_of::<u32>();
+        let bytes_per_invalidation = size_of::<u128>() + size_of::<u128>();
```

Comment:
let's have this as a static variable, and reuse it. I see it declared in multiple places. if we forget it will be a bug